locales/client/cn.json

```diff
@@ -945,13 +945,14 @@
         }
     },
     "ISCNARWidget": {
-    "transaction": {
-      "firstStep": "签署  (1/2)",
-      "secondStep": "签署  (2/2)",
-      "reminder": "请以你的钱包签署",
-      "retry": "重试",
-      "doNotCloseReminder": "请勿关闭此视窗",
-      "signAgain": "你还需要再签署一次才完成程序"
+        "transaction": {
+            "firstStep": "签署  (1/2)",
+            "secondStep": "签署  (2/2)",
```

Comment:
Why there is extra space before the bracket?

Reply:
No idea, can remove